config/500-webhook-configuration.yaml

```diff
@@ -32,7 +32,17 @@ webhooks:
   failurePolicy: Fail
   sideEffects: None
   timeoutSeconds: 25
-
+  rules:
```

Comment:
These get overwritten, so these won't have any effect. You need to add these here:
https://github.com/sigstore/policy-controller/blob/main/cmd/webhook/main.go#L125

And might have to add a new one for Pod that supports the `ephemeralcontainers` subresource here:
https://github.com/sigstore/policy-controller/blob/main/cmd/webhook/main.go#L112

Reply:
gotcha! Thanks for the feedback @vaikas :)

Reply:
Fixed now 🎉 !